src/components/Product/LLMAnalytics/index.tsx

```diff
@@ -0,0 +1,386 @@
+        description: 'Understand how user feedback translates into engagement',
+        image: <StaticImage src="./images/question-types.png" width={428} placeholder="none" />,
+    },
+]
+
+const subfeaturesItemCount = 2
+const subfeatures = [
+    {
+        title: 'Dashboard templates',
+        description: 'Get instant insights with ready-made templates',
+        icon: <IconBolt />,
+    },
+    {
+        title: 'Simple integrations',
+        description: 'Integrate seamlessly with Langfuse or Helicone',
+        icon: <IconPeople />,
+    },
+]
+
+const questions = [
+    {
+        question: 'Are there generation latency spikes?',
+    },
+    {
+        question: 'Does interacting with the LLM feature relate to higher retention rates?',
+    },
+    {
+        question: 'How does the LLM feature impact my conversion rates?',
+    },
+    {
+        question: 'What are my LLM costs by customer, model, and in total?',
+    },
+]
+
+const pairsWithItemCount = 3
+const PairsWithArray = [
+    {
+        icon: <IconGraph />,
+        product: 'Product analytics',
+        description: 'Use insights to dive deeper into your data and track sign-ups, payments, and more.',
+        url: '/product-analytics',
+    },
+    {
+        icon: <IconMessage />,
+        product: 'Surveys',
+        description: 'Target users with surveys to gather CSAT scores and measure satisfaction with your model.',
+        url: '/surveys',
+    },
+    {
+        icon: <IconRewindPlay />,
+        product: 'Session replay',
+        description: 'Watch recordings of users interacting with your LLM to get full context about their behavior.',
+        url: '/session-replay',
+    },
+]
+
+export const LLMAnalytics = () => {
+    const { elevenlabs } = useStaticQuery(graphql`
+        fragment ProductCustomerFragment on Mdx {
+            fields {
+                slug
+            }
+            frontmatter {
+                logo {
+                    publicURL
+                }
+                logoDark {
+                    publicURL
+                }
+            }
+        }
+        {
+            elevenlabs: mdx(slug: { eq: "customers/elevenlabs" }) {
+                ...ProductCustomerFragment
+            }
+        }
+    `)
+    const { fullWidthContent } = useLayoutData()
+    return (
+        <>
+            <SEO
+                title="LLM analytics - PostHog"
+                description="Get instant insights into LLM products and model performance"
+                image={`/images/og/surveys.jpg`}
+            />
+            <div className={`${fullWidthContent ? 'max-w-full px-8' : 'max-w-7xl mx-auto'} px-5 py-10 md:pt-20 pb-0`}>
+                <Hero
+                    color="yellow"
+                    icon={<IconAdvanced />}
+                    beta={true}
+                    product={product.capitalized}
+                    title='Unlock <span class="text-red dark:text-yellow">instant insights</span> for LLM products'
```

Comment:
Something about this heading doesn't do it for me. The emphasis is on "instant insights", but really want to emphasize LLM products

What if we remove the orange highight, and just make the text "Unlock insights for LLM products" ?

Reply:
```suggestion
                    title='Unlock instant insights for <span class="text-red dark:text-yellow">LLM products</span>'
```